features/fixtures/mazerunner/app/src/main/java/com/bugsnag/android/Flush.java

```diff
@@ -0,0 +1,8 @@
+package com.bugsnag.android;
+
+public class Flush {
+    public void flush() {
```

Comment:
This class should be named more clearly, and the `flush` method can be `static` so that we don't need to create object instances to call `flush()`

```suggestion
public class BugsnagInternals {
    private BugsnagInternals() {}
    
    public static void flush() {
```